cassandra/datastax/cloud/__init__.py

```diff
@@ -165,3 +166,17 @@ def _ssl_context_from_cert(ca_cert_location, cert_location, key_location):
     ssl_context.load_cert_chain(certfile=cert_location, keyfile=key_location)
 
     return ssl_context
+
+
+def _pyopenssl_context_from_cert(ca_cert_location, cert_location, key_location):
+    try:
+        from OpenSSL import SSL
+    except ImportError:
+        return None
+    ssl_context = SSL.Context(SSL.TLSv1_METHOD)
+    ssl_context.set_verify(SSL.VERIFY_PEER, callback=lambda _1, _2, _3, _4, ok: ok)
```

Comment:
sorry, still trying to understand the set_verify use. Maybe a good thing to summarize set_verify vs info_callback in the PR summary.